src/components/calculator/PersonRiskControls.tsx

```diff
@@ -193,10 +202,14 @@ export const PersonRiskControls: React.FunctionComponent<{
           showTooltip={true}
           useHoverDesc={false}
         />
-      ) : null}
-      <br />
-    </React.Fragment>
-  )
+      )
+    }
+    return (
+      <div>No vaccine data available for this region.</div>
```

Comment:
I would say "Since our system doesn't get have vaccine data ifor your region, we aren't able to provide a way for you to adjust your score based on whether or not the person you're seeing is vaccinated."

Reply:
"We can't factor in other people's vaccination statuses because we don't have vaccine data for your region."